### PR TITLE
Fpo obs project structure

### PR DESCRIPTION
**Pull request by Koufan-De-King**
Refactored structure of OBS module

### CODE REVIEW

File: obs/obs-rest-api/pom.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    </dependencies>
+
+<!--    <build>-->
+<!--        <plugins>-->
+<!--            <plugin>-->
+<!--                <groupId>org.springframework.boot</groupId>-->
+<!--                <artifactId>spring-boot-maven-plugin</artifactId>-->
+<!--            </plugin>-->
+<!--        </plugins>-->
+<!--    </build>-->
```
Comment (Arielpetit) [inline obs/obs-rest-api/pom.xml:64]: Hello @Koufan-De-King this dont have to be here, can you please remove that?

File: obs/obs-rest-server/pom.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    </dependencies>
+
+<!--    <build>-->
+<!--        <plugins>-->
+<!--            <plugin>-->
+<!--                <groupId>org.springframework.boot</groupId>-->
+<!--                <artifactId>spring-boot-maven-plugin</artifactId>-->
+<!--            </plugin>-->
+<!--        </plugins>-->
+<!--    </build>-->
```
Comment (Arielpetit) [inline obs/obs-rest-server/pom.xml:56]: Same here

File: obs/obs-service-api/pom.xml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+<!--                                    </excludes>-->
+<!--                                </bannedDependencies>-->
+<!--                            </rules>-->
+<!--                            <fail>true</fail>-->
+<!--                        </configuration>-->
+<!--                    </execution>-->
+<!--                </executions>-->
+<!--            </plugin>-->
+<!--        </plugins>-->
+<!--    </build>-->
```
Comment (Arielpetit) [inline obs/obs-service-api/pom.xml:68]: Same here

File: obs/obs-service-impl/pom.xml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    </dependencies>
+
+<!--    <build>-->
+<!--        <plugins>-->
+<!--            <plugin>-->
+<!--                <groupId>org.springframework.boot</groupId>-->
+<!--                <artifactId>spring-boot-maven-plugin</artifactId>-->
+<!--            </plugin>-->
+<!--        </plugins>-->
+<!--    </build>-->
```
Comment (Arielpetit) [inline obs/obs-service-impl/pom.xml:64]: Same here

File: obs/pom.xml [hunk 5]
```diff
@@ -0,0 +1,379 @@
...
+                <groupId>org.junit.jupiter</groupId>
+                <artifactId>junit-jupiter</artifactId>
+                <version>${junit-jupiter-api.version}</version>
+                <scope>test</scope>
+            </dependency>
+
+        </dependencies>
+    </dependencyManagement>
+
+</project>
```
Comment (Arielpetit) [inline obs/pom.xml:379]: I think this should be in the pom of the webank-online banking itself not here

File: online-banking-app/src/test/java/com/adorsys/webank/OnlineBankingApplicationTests.java [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+//
+//@SpringBootTest(classes = OnlineBankingApplication.class) // Specify main class
+//public class OnlineBankingApplicationTests {
+//
+//	@Test
+//	public void contextLoads() {
+//		// This test intentionally does nothing.
+//		// The @SpringBootTest annotation simulates application startup.
+//	}
+//}
```
Comment (Arielpetit) [inline online-banking-app/src/test/java/com/adorsys/webank/OnlineBankingApplicationTests.java:15]: I think you forgot to uncomment the test

**Review by Arielpetit — APPROVED** [review]

**Review by NkwaTambe — APPROVED** [review]

**Review by Elwizzy12 — DISMISSED** [review]

**Review by nancymuyeh — DISMISSED** [review]

**Review by Arielpetit — APPROVED** [review]
Looks good, great job

**Review by Elwizzy12 — APPROVED** [review]

**Pull request merged**